src/components/header/navMenu.js

```diff
@@ -1,37 +1,54 @@
 import React from 'react';
-import { NavLink } from 'react-router-dom';
+import { NavLink, withRouter } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
 import styles from './navMenu.module.scss';
 
-const NavMenu = ({ selectedTeam: { teamColor, urlName }, playerId }) => {
-  const activeStyle = {
-    color: teamColor,
+const NavMenu = ({
+  history: {
+    location: { pathname },
+  },
+  selectedTeam: { teamColor, urlName },
+  playerId,
+}) => {
+  const activeBorder = {
     borderBottom: `4px solid ${teamColor}`,
   };
 
+  const getActiveStyle = (path) => (pathname.indexOf(path) >= 0 ? activeBorder : null);
+
   return (
     <div className={styles.container}>
       <div className={styles.linkContainer}>
```

Comment:
No need for linkContainer anymore, remove those divs throughout this component and the `.linkContainer` css